package.json

```diff
@@ -13,6 +13,7 @@
     "@babel/plugin-transform-arrow-functions": "^7.12.1",
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-react": "^7.12.10",
+    "@emotion/react": "^11.1.5",
```

Comment:
for some reason my branch won't compile without these additional devDependencies

Reply:
Still the case without the additional select2 dependency? I would expect that no changes/additions to package.json/lock would be required.

Reply:
indeed. fixed